src/datoso/helpers/__init__.py

```diff
@@ -115,7 +120,10 @@ def remove_folder(path):
     def remove(path):
         """ Remove file or folder. """
         try:
-            os.unlink(path)
+            if os.path.isdir(path):
```

Comment:
Same as Before, except here we can remove all try/catch and change function to simple validations.
just adding at the beginning of the function
```
if not os.path.exists(path):
   return
```